spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/pubsub/GcpPubSubEmulatorConfigurationTests.java

```diff
@@ -149,10 +153,9 @@ public void testSubscriberFlowControlSettings() {
 		this.contextRunner.run((context) -> {
 			FlowControlSettings settings = context
 					.getBean("subscriberFlowControlSettings", FlowControlSettings.class);
-			Assert.assertEquals(17, (long) settings.getMaxOutstandingElementCount());
-			Assert.assertEquals(18, (long) settings.getMaxOutstandingRequestBytes());
-			Assert.assertEquals(LimitExceededBehavior.Ignore,
-					settings.getLimitExceededBehavior());
+			assertThat((long) settings.getMaxOutstandingElementCount()).isEqualTo(17);
```

Comment:
Don't need a cast in these two lines; AbstractLongAssert has an overridden version that takes a primitive long.